src/main/java/se/bjurr/violations/comments/bitbucketserver/lib/BitbucketServerCommentsProvider.java

```diff
@@ -88,7 +88,12 @@ public void createCommentWithAllSingleFileComments(final String comment) {
   @Override
   public void createSingleFileComment(
       final ChangedFile file, final Integer line, final String comment) {
-    client.pullRequestComment(file.getFilename(), line, comment);
+    final BitbucketServerComment bitbucketComment =
+        client.pullRequestComment(file.getFilename(), line, comment);
+
+    if (violationCommentsToBitbucketApi.getCreateSingleFileCommentsTasks()) {
```

Comment:
You are changing `client.pullRequestComment` to return the comment and not actually create the comment.

If this is `false` then there will be no comment created at all right? Commenting single files, without creating a task, no longer works with this change?

Reply:
You are right! Merging it now.